### ISSUE TITLE
WimgApi.CopyFile using a callback

### ISSUE THREAD

**Issue by King-GamesRG** [josemesona/ManagedWimgApi#13]
WimgApi.CopyFile using a callback fails with following exception:

```
Error copying WIM file to O:\ : System.OverflowException: TimeSpan overflowed because the duration is too long.
   at System.TimeSpan.Interval(Double value, Int32 scale)
   at Microsoft.Wim.CopyFileProgress.CopyProgressHandler(UInt64 totalFileSize, UInt64 totalBytesTransferred, UInt64 streamSize, UInt64 streamBytesTransferred, UInt32 streamNumber, UInt32 callbackReason, IntPtr sourceFile, IntPtr destinationFile, IntPtr data)
   at Microsoft.Wim.WimgApi.NativeMethods.WIMCopyFile(String pszExistingFileName, String pszNewFileName, CopyProgressRoutine pProgressRoutine, IntPtr pvData, Boolean& pbCancel, UInt32 dwCopyFlags)
   at Microsoft.Wim.WimgApi.CopyFile(String sourceFile, String destinationFile, WimCopyFileOptions options, CopyFileProgressCallback copyFileProgressCallback, Object userData)
   at StickCreatorGui.MainWindow.CopyWimToDirectory() in C:\xxxxx
```

**Comment by josemesona** [comment]
Thanks for reporting this, it looks like a behavior change in the WIMGAPI.  I was able to repro it and will send a pull request to fix it.

**Comment by King-GamesRG** [comment]
will you update the gunet package as well?

**Comment by jeffkl** [comment]
Yes, I was waiting until https://github.com/josemesona/ManagedWimgApi/pull/23 was done and then I'll push a new package

**Comment by josemesona** [comment]
The fix is available in 1.2.3: https://www.nuget.org/packages/Microsoft.Wim/1.2.3
